applications/convection_diffusion/rotating_hill/tests/convective_implicit_time_int_amr.output

```diff
@@ -0,0 +1,782 @@
+
+Numerical parameters:
+  Use cell-based face loops:                 false
+  Store velocity in DoF vector:              false
+  Use over-integration:                      false
+
+Generating grid for 2-dimensional problem:
+
+  Max. number of refinements:                3
+  Number of cells:                           64
+
+Construct convection-diffusion operator ...
+
+Discontinuous Galerkin finite element discretization:
+
+  degree of 1D polynomials:                  4
+  number of dofs per cell:                   25
+  number of dofs (total):                    1600
+
+... done!
+
+Setup convection-diffusion operator ...
+
+... done!
+
+Setup Multistep time integrator ...
+
+Calculation of time step size according to CFL condition:
+
+  CFL:                                       2.5000e-01
+  Time step size (CFL global):               6.8342e-04
+
+Adjust time step size to hit end time:
+
+  Time step size:                            6.8306e-04
+
+... done!
+
+Starting time loop ...
+
+Calculate error for all fields at time t = 0.0000e+00:
+  Relative error (L2-norm): 4.32038e-02
+
+________________________________________________________________________________
+
+ Time step number = 1       t = 0.00000e+00 -> t + dt = 6.83060e-04
+________________________________________________________________________________
+
+Discontinuous Galerkin finite element discretization:
+
+  degree of 1D polynomials:                  4
+  number of dofs per cell:                   25
+  number of dofs (total):                    1375
+
+Setup convection-diffusion operator ...
+
+... done!
+
+Discontinuous Galerkin finite element discretization:
+
+  degree of 1D polynomials:                  4
+  number of dofs per cell:                   25
+  number of dofs (total):                    1300
+
+Setup convection-diffusion operator ...
+
+... done!
+
+Discontinuous Galerkin finite element discretization:
+
+  degree of 1D polynomials:                  4
+  number of dofs per cell:                   25
+  number of dofs (total):                    1300
+
+Setup convection-diffusion operator ...
+
+... done!
+
+Calculate error for all fields at time t = 5.0546e-02:
+  Relative error (L2-norm): 4.31811e-02
+
+________________________________________________________________________________
+
+ Time step number = 75      t = 5.05464e-02 -> t + dt = 5.12295e-02
+________________________________________________________________________________
+
+Discontinuous Galerkin finite element discretization:
+
+  degree of 1D polynomials:                  4
+  number of dofs per cell:                   25
+  number of dofs (total):                    1375
+
+Setup convection-diffusion operator ...
+
+... done!
+
+Discontinuous Galerkin finite element discretization:
+
+  degree of 1D polynomials:                  4
+  number of dofs per cell:                   25
+  number of dofs (total):                    1450
+
+Setup convection-diffusion operator ...
+
+... done!
```

Comment:
I think we can improve this output, i.e. it looks like one can currently not extract from the output that AMR is performed.

Reply:
In order to not make the output too lengthy, a one-liner per AMR step should be enough in my opinion.

Reply:
Ok, I will do something like "Performing adaptive mesh refinement: Number of dofs (total): ...". So **that** one line basically.

Reply:
Hmmmm ... I introduced now `print_discretization_info` that differentiates these cases. In ExaDG, we frequently have some "Doing something" "... done!" output, but that was not the case here, so I pulled this out of the `initialize_dof_handler_and_constraints`.
The setup now has some `bool const verbosity = true`, which regulates the "output depth". Honestly I do not like this too much, should we just skip the "Setup conv-diff op." "...done!" for the case when AMR is enabled?
Please check 
https://github.com/exadg/exadg/pull/586/commits/18bbf4d09c111951be71f4c1d9254edf5356b466

Reply:
I agree that we do not want to have "Setup conv-diff op." "...done!" during timestepping with AMR.